src/mon/MDSMonitor.cc

```diff
@@ -1758,7 +1758,8 @@ int MDSMonitor::management_command(
     // Persist the new FSMap
     pending_fsmap.filesystems[new_fs->fscid] = new_fs;
     return 0;
-  } else if (prefix == "fs set_default") {
+  } else if (prefix == "fs set_default" ||
```

Comment:
probably we can update `test/pybind/test_ceph_argparse.py` accordingly to `s/set_default/set-default/` to remove another hurdle if we want to deprecate `fs set_default` in future.